src/shell/braces.zig

```diff
@@ -803,4 +795,32 @@ pub fn NewLexer(comptime encoding: Encoding) type {
     };
 }
 
-const assert = bun.assert;
+const t = std.testing;
+test Lexer {
+    var arena = std.heap.ArenaAllocator.init(t.allocator);
+    defer arena.deinit();
+
+    const TestCase = std.meta.Tuple(&[_]type{ []const u8, []const Token });
```

Comment:
```suggestion
    const TestCase = struct { []const u8, []const Token };
```

or name the fields if you want

Reply:
Is `Tuple` deprecated?

Reply:
I don't know about `Tuple` specifically, but my understanding is all of `std.meta` is on thin ice. In this case I also think the code would be nicer as a struct with named fields instead of a tuple.

Reply:
switched to `struct`